.golangci.yml

```diff
@@ -1,5 +1,5 @@
 run:
-  deadline: 5m
+  timeout: 15m
```

Comment:
- According to https://github.com/golangci/golangci-lint/pull/793, "deadline" is deprecated, so change it to "timeout".
-  If there is no cache, the command may exceed 5 minutes, so extend the timeout period to 15minutes.